README.md

```diff
@@ -47,6 +47,33 @@ WebNN requires a compatible browser to run, and Windows 11 v21H2 (DML 1.6.0) or
 4. In the drop-down menu, select `Enabled`. 
 5. Relaunch your browser.
 
+#### Running WebNN on NPU
+At present, the [image classification](https://webmachinelearning.github.io/webnn-samples/image_classification/) and [object detection](https://webmachinelearning.github.io/webnn-samples/object_detection/) samples support NPU.
+
+* Install NPU driver:
+
+  For Intel® AI Boost NPU of Intel® Core™ Ultra processors, download and install the [latest Intel NPU driver](https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html).
+
+* Google Chrome Canary:
+1. Download the latest redistributable [Microsoft.AI.DirectML](https://www.nuget.org/packages/Microsoft.AI.DirectML/).
+2. Extract the package and copy the DirectML.dll (e.g. \bin\x64-win\DirectML.dll) to the Chrome Canary’s directory (e.g. C:\Users\"username"\AppData\Local\Google\Chrome SxS\Application\"version number").
+3. Launch your browser in Windows Command Line:
+```bash
+"C:\Users\\"username"\AppData\Local\Google\Chrome SxS\Application\chrome.exe" --use-redist-dml --disable_webnn_for_npu=0
+```
+
+* Microsoft Edge Canary:
+1. Launch your browser in Windows Command Line:
+```bash
+"C:\Users\\"username"\AppData\Local\Microsoft\Edge SxS\Application\msedge.exe" --disable_webnn_for_npu=0
```

Comment:
Do the same for edge command line.

Reply:
If use `"%LOCALAPPDATA%\Microsoft\Edge SxS\Application\msedge.exe" --use-redist-dml --disable_webnn_for_npu=0 --disable-gpu-sandbox` , and if there is no directml.dll in edge version folder, it will load system directml.dll whose version is not enough to run WebNN NPU. If we don't mention copy directml.dll into edge version folder, we can't use `--use-redist-dml`, wdyt?